src/routes/healthcheck/healthcheck-router.ts

```diff
@@ -14,4 +14,6 @@ HealthcheckRouter.get("/deepcheck", DeepcheckGet);
 /**
  * healthcheck endpoint to check that the app started properly
  */
-HealthcheckRouter.get("/healthcheck", HealthcheckGet);
+HealthcheckRouter.get("/healthcheck", HealthcheckGetPost);
+// To troubleshoot connectivity between GHEs and the app
+HealthcheckRouter.post("/healthcheck/:uuid", HealthcheckGetPost);
```

Comment:
Care the reason of POST instead of GET?

Reply:
because GHEs sending post requests

Reply:
Okay, you need POST in poco settings as well

Reply:
oo good catch, and path by the looks

Reply:
Can't we just have:
`HealthcheckRouter.get("/healthcheck/:uuid?", HealthcheckGetPost);`

Reply:
Can we chain then?

```
HealthcheckRouter.route("/healthcheck/:uuid?")
  .get(HealthcheckGetPost)
  .post(HealthcheckGetPost);
 ```

Reply:
so. many responses on the pr, haha, #TeamWork!!

Reply:
I'd prefer to keep it simple :)

Reply:
Makes the dream work.